docs/quick-start.md

```diff
@@ -0,0 +1,105 @@
+# Freshmint Quick Start Guide
+
+This guide covers how to use the Freshmint CLI tool to create and manage an NFT application on Flow.
+
+## Installation
+
+```sh
+npm install -g freshmint
+```
+
+:warning: **Important:** Freshmint depends on the Flow CLI, so make sure you have it installed:
+
+- [Install the Flow CLI](https://developers.flow.com/tools/flow-cli/install)
+
+## Create a project
+
+Start by creating a new Freshmint project:
+
+```sh
+fresh start your-project-name
+
+cd ./your-project-name
+```
+
+This command creates a new directory with the following:
+
+- A project configuration file (`freshmint.yaml`).
+- A custom NFT contract, just for you!
+- Transactions and scripts to interact with your contract.
+- Sample NFTs and image assets.
+- A pre-built Next.js web app for running a live drop.
+
```

Comment:
- Flow configuration files.
- A file to define the pinning service endpoint and key as well as testnet or mainnet account information (`.env`).